src/component/PricingButton/PricingButton.jsx

```diff
@@ -37,16 +48,31 @@ const PricingButton = ({
   originAddress,
   destinationAddress,
 }) => {
+
+  function clearDisplay() {
+    const default_display = {
+      error: true,
+      status: undefined,
+      message: {
+        lyft: undefined,
+        uber: undefined,
+      },
+    }
+    changeDisplayPricing(default_display);
+  }
+
```

Comment:
Again, this is redudant. Since we are not doing anything with the object other than calling `changeDisplayPricing`, we can simply do: 
```suggestion
    changeDisplayPricing({
      error: true,
      status: undefined,
      message: {
        lyft: undefined,
        uber: undefined,
      },
    });
  }
```
The magic of React